code/_globalvars/lists/mobs.dm

```diff
@@ -48,24 +48,103 @@ GLOBAL_LIST_EMPTY(roundstart_species)
 GLOBAL_LIST_EMPTY_TYPED(xeno_caste_datums, /list/datum/xeno_caste)
 GLOBAL_LIST_INIT(all_xeno_types, list(
 	/mob/living/carbon/xenomorph/runner,
+	/mob/living/carbon/xenomorph/runner/mature,
+	/mob/living/carbon/xenomorph/runner/elder,
```

Comment:
why doesnt this just use a typecache

Reply:
The biggest problem is that you can spawn non-admeme xenos in Valhalla if you did a strict typecache, as well as xenos that could be used to grief/lag the server like xenos with functional AI.